python/tests/backends/test_braket.py

```diff
@@ -289,7 +289,51 @@ def kernel(qubit_count: int):
     assert "0000" in counts
     assert "1111" in counts
 
+def test_kernel_subveqs():
 
+    @cudaq.kernel
+    def kernel():
+        qreg = cudaq.qvector(4)
+        x(qreg[1])
+        x(qreg[2])
+        v = qreg[1:3]
+        mz(v)
+
+    counts = cudaq.sample(kernel, shots_count=100)
+    assert len(counts) == 1
+    assert "11" in counts
+
+def test_kernel_two_subveqs():
+
+    @cudaq.kernel
+    def kernel():
+        qreg = cudaq.qvector(4)
+        x(qreg[1])
+        x(qreg[2])
+        v = qreg[0:2]
+        v = qreg[2:3]
+        mz(v)
+
+    counts = cudaq.sample(kernel, shots_count=100)
+    assert len(counts) == 1
+    assert "011" in counts
```

Comment:
Q: I don't understand what this test is checking since `v` was overwritten to be of size 2...?

Reply:
I agree w/ Pradnya. As written, I would expect this to be `assert "1" in counts` because I would expect it to be showing the value of `qreg[2]`.

Reply:
thanks, it is missing a meazurement!

Reply:
updated